imports/lib/models/document_permissions.ts

```diff
@@ -5,7 +5,7 @@ import Base from './base';
 const DocumentPermissions = new Base<DocumentPermissionType>('document_perms');
 DocumentPermissions.attachSchema(DocumentPermissionSchema);
 DocumentPermissions.publish(function (this: Subscription, q) {
-  return { ...q, user: this.userId };
+  return { ...q, user: this.userId! };
```

Comment:
I don't think we have anything that actually prevents this from being `undefined/null` if these subs get requested by a connection where no user has logged in?